### ISSUE TITLE
About "original_logits" and "outputs" issues

### ISSUE THREAD

**Issue by konioyxgq** [HuangYG123/CurricularFace#20]
When calculating top1 and top5, why use "**original_logits**" instead of "**outputs**"? Although this does not affect the training process, I found that the top1 obtained with "**original_logits**" will be much larger than the top1 obtained with "**outputs**". This is exactly the opposite of my expectations. Why is that?

**Comment by CloudWalking0** [comment]
> 
> 
> When calculating top1 and top5, why use "**original_logits**" instead of "**outputs**"? Although this does not affect the training process, I found that the top1 obtained with "**original_logits**" will be much larger than the top1 obtained with "**outputs**". This is exactly the opposite of my expectations. Why is that?

Hello! 
    I have the same problem with you. Why using "original logits" instead of "outputs" during calculating? Have you already known the answer?  Thx :)
